server/models/Profile.js

```diff
@@ -0,0 +1,45 @@
+import mongoose from 'mongoose';
+const mongoose = require("mongoose")
+
+const DAYS = ["Mon", "Tue", "Wed", "Thu", "Fri", "Sat", "Sun"]
+
+const profileSchema = new mongoose.Schema({
+    first_name: {
+        type: String,
+        required: true,
+    },
+    last_name: {
+        type: String,
+        required: true,
+    },
+    description: {
+        type: String,
+        required: true,
+    },
+    availability: [{
+            type: String,
+            enum: DAYS,
+        }],
+    gender: {
+        type: String,
+    },
+    birth_date: {
+        type: Date
+    },
+    email_address: {
+        type: String
+    },
+    phone_numbers: [{
+        type: String
+    }],
+    address: {
+        type: String
+    },
+    user: {
+        type: mongoose.Schema.Types.ObjectId,
+        ref: "User"

```

Comment:
```suggestion
        ref: "User"
        required: true
```
This should be a required attribute as we would want to specify the owner of a profile when we create one

Reply:
This was fixed in another branch